spec/ddtrace/contrib/rails/support/rails4.rb

```diff
@@ -6,6 +6,9 @@
   require 'ddtrace/contrib/sidekiq/server_tracer'
 end
 
+# for log_injection testing
```

Comment:
Are we able to extract these changes to `spec/ddtrace/contrib/rails/support/rails*.rb` into `spec/ddtrace/contrib/rails/support/base.rb` or `spec/ddtrace/contrib/rails/support/application.rb`?

Normally we'd like these `rails*.rb` files to be only for changes specific for that version (even though the file might not be that clean today).

Reply:
was able to move nearly all of this into `base`, with the exception of some cleanup code that lives in each railsX file.